src/AuthorizedSocketConnection.js

```diff
@@ -176,9 +176,9 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
         document,
         null,
         {
-          subscribe: (...args) =>
+          subscribe: async (...args) =>
             AsyncUtils.filter(
-              this.config.subscriber.subscribe(...args),
+              await this.config.subscriber.subscribe(...args),
```

Comment:
i think this is a little easier to read if you pull out the `await`. something like:

```js
subscribe: async (...args) => {
  const stream = await this.config.subscriber.subscribe(...args);
  return AsyncUtils.filter(stream, this.isAuthorized);
},
```

i know it's the same, but i had to read this like 3 times to figure out what it was doing